builtin/providers/vsphere/resource_vsphere_virtual_machine.go

```diff
@@ -740,7 +770,7 @@ func resourceVSphereVirtualMachineDelete(d *schema.ResourceData, meta interface{
 	return nil
 }
 
-func waitForNetworkingActive(client *govmomi.Client, datacenter, name string) resource.StateRefreshFunc {
+func waitForNetworkingActive(client *govmomi.Client, datacenter, name string, waitForNetworkInterfaces bool) resource.StateRefreshFunc {
```

Comment:
Please update this PR to use this.  We will also need a unit test 😄  Pretty please.